app/views/facility_orders/_merge_order_form.html.haml

```diff
@@ -31,11 +32,19 @@
           input_html: { class: "js--chosen" },
           include_blank: false,
           label: OrderDetail.human_attribute_name(:order_status)
-        = f.input :account_id,
-          collection: @add_to_order_form.available_accounts,
-          include_blank: !@order.account.active?,
-          hint: @order.account.active? ? "" : t(".original_account_inactive", account: @order.account),
-          input_html: { class: "js--chosen" }
+        - if enable_cross_core_changes
+          = f.input :account_id,
+            collection: @add_to_order_form.available_accounts,
+            hint: @order.account.active? ? "" : t(".original_account_inactive", account: @order.account),
+            include_blank: !@order.account.active?,
+            input_html: { class: "js--edit-order__account js--chosen",
```

Comment:
If this line is the only difference between the two cases, maybe we can set that value separately like:
```
input_html = enable_cross_core_changes ? { class: "js--edit-order__account js--chosen", data: { "include-blank" => !@order.account.active? } } : { class: "js--chosen" }
```